nft_ingester/src/config.rs

```diff
@@ -213,13 +214,12 @@ pub struct IngesterClapArgs {
     pub big_table_config: Option<BigTableConfig>,
 
     #[clap(
-        long("disable-bubblegum-backfiller"),
-        action = ArgAction::SetFalse,
+        long,
         env = "RUN_BUBBLEGUM_BACKFILLER",
-        default_value_t = true,
-        help = "#bubbl Disable bubblegum backfiller (default: true)"
+        default_value = "true",
```

Comment:
Hm. How `default_value` can be `true` if it is `Option`? Shouldn't it be then Some(true)? Do we need an option?

Reply:
Using Option is only one case to set default bool value in true. So it's just a workaround to solve a Clap issue